ElementX/Sources/Screens/RoomScreen/View/TimelineTableViewController.swift

```diff
@@ -168,6 +170,13 @@ class TimelineTableViewController: UIViewController {
         guard !hasAppearedOnce else { return }
         scrollToBottom(animated: false)
         hasAppearedOnce = true
+        if ServiceLocator.shared.settings.timelineDiffableAnimationsEnabled {
+            // This is to prevent the SS proxy issue that forces a full refresh of all the timeline items
+            // the first time the timeline is opened
+            DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
+                self.shouldAnimate = true
+            }
+        }
```

Comment:
```suggestion
            // This is to prevent the SS proxy issue that forces a full refresh of all the timeline items
            // the first time the timeline is opened
            DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
                self.shouldAnimate = ServiceLocator.shared.settings.timelineDiffableAnimationsEnabled
            }
```

Can explain me better why the need to dispatch after 2 seconds?

Reply:
yeah there is a proxy issue... essentially sometimes (not always) when you enter in the timeline of a room, we confirm that the existing messages with the server by receiving them again. However since there is a proxy issue with the txn_id what happens is that the sdk does not recognize such items, and just completely flushes the existing timeline and regenerates all of them, with a new identifier (so they are essentially considered new messages) which of course removes existing messages and replaces with new ones that exactly look the same creating a blinking animation

Reply:
Good news btw Jonas has found an SDK fix for this issue, so I am probably removing this delay in the next SDK update.